src/common/reconciler_test.go

```diff
@@ -16,9 +16,8 @@ import (
 	"github.com/rocktavious/autopilot/v2023"
 )
 
-// TODO: tests here would be much easier to use if we had a builder for service registrations
-// TODO: tests here would be much easier to use if we had a builder for existing services
-// TODO: tests here would be much easier to use if we had a builder for clients so that we don't have to `panic("should not be called")` everything.
+// TODO: use builder pattern for client so that every function that isn't explicitly stubbed will panic
```

Comment:
I added this TODO - we could probably cut the lines of code in this file by 40% by using a builder pattern.

I don't think a builder for service/service registration is necessary, so those lines are removed.

In progress: https://github.com/OpsLevel/kubectl-opslevel/pull/283